src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Frameworks/SupportedTargetFrameworksProvider.cs

```diff
@@ -9,7 +9,7 @@
 using System.Threading.Tasks.Dataflow;
 using Microsoft.Build.Framework.XamlTypes;
 using Microsoft.VisualStudio.ProjectSystem.Properties;
-
```

Comment:
Nit: Maybe preserve the blank line?

Reply:
We removed blank lines between `using` groups a while back. I assumed that also applied between namespace and alias directives.